HISTORY.md

```diff
@@ -0,0 +1,84 @@
+# Looking Back in Time at BackInTime
+
+*by Michael Büker, 2024*
+
+The history of the BackInTime project, which at the time of this writing already spans nearly 16 years, is best understood in four periods:
+
+1. The **First Era** from 2008 to 2012, releases 0.5 to ~1.0.12
+2. The **Second Era** from 2012 to 2019, releases ~1.0.14 to 1.2
+3. A **Dark Age** from 2019 to 2022, releases 1.2.0 to 1.3.2
```

Comment:
"Dark Age" is funny but IMHO sounds a bit too theatrical for a document like this. Could be a bit more serious.